test/lenses.js

```diff
@@ -41,16 +56,24 @@ describe('view, over, and set', function() {
 
   they('may be applied to composed lenses', function() {
     var streetLens = R.compose(addressLens, headLens);
+    var dogLens = R.compose(R.lensPath(['pets']), R.lensPath(['dog']));
+
+    eq(R.view(dogLens, alice), R.view(R.lensPath(['pets', 'dog']), alice));
```

Comment:
No, I hadn't caught that when lenses were introduced, or when lensProp was introduced.  We do need to fix this, I believe.  I think that the unit test suite is also part of our documentation.

But don't bother with it in this PR.  I hadn't noticed it before now.